themes/10up-theme/includes/blocks.php

```diff
@@ -42,19 +42,49 @@ function setup() {
 }
 
 /**
- * Add in blocks that are registered in this theme
+ * Automatically registers all blocks that are located within the includes/blocks directory
  *
  * @return void
  */
 function register_theme_blocks() {
 	// Filter the plugins URL to allow us to have blocks in themes with linked assets. i.e editorScripts
 	add_filter( 'plugins_url', __NAMESPACE__ . '\filter_plugins_url', 10, 2 );
 
-	// Require custom blocks.
-	require_once TENUP_THEME_BLOCK_DIR . '/example-block/register.php';
+	// Register blocks for the active theme, for both parent and child theme,
+	// if applicable.
+	foreach ( wp_get_active_and_valid_themes() as $theme ) {
+		$block_directory_path = $theme . '/includes/blocks/';
+		if ( file_exists( $block_directory_path ) ) {
+			$block_json_files = glob( $block_directory_path . '*/block.json' );
```

Comment:
Is there a reason you are attempting to read the locations instead of using the constants defined in the theme?

https://github.com/10up/wp-scaffold/blob/trunk/themes/10up-theme/functions.php#L15

Reply:
The only reason there is that right here we are looping over child themes as well. 

This is why it is being build out of the theme loop here:
```php
foreach ( wp_get_active_and_valid_themes() as $theme ) {
```